lib/ansible/modules/network/aos/aos_ip_pool.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/python
+#
+# (c) 2017 Apstra Inc, <community@apstra.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: aos_ip_pool
+author: damien@apstra.com (@dgarros)
+version_added: "2.3"
+short_description: Manage AOS IP Pool
+description:
+  - Used to add an IP Pool to AOS-server from playbook
+  - Used to add an IP Pool to AOS-server from a JSON file
+  - Used to delete an IP Pool
+  - Used to gather an IP Pool
+requirements:
+  - aos-pyez
+options:
+  session:
+    description:
+      - An existing AOS session as obtained by aos_login module
```

Comment:
Once aos_login has been merged you can change this to `M(aos_login)` to generate a hyperlink to the Module docs for aos_login.